usr/libexec/anon-ws-disable-stacked-tor/systemd-unit-files-generator

```diff
@@ -115,6 +116,10 @@ else
       ## IP HARDCODED. Set GATEWAY_IP through a drop-in configuration snippet in /etc/anon-ws-disable-stacked-tor.d configuration folder instead.
       GATEWAY_IP="10.152.152.10"
    fi
+   if [ "$GATEWAY_IP6" = "" ]; then
+      ## IP HARDCODED. Set GATEWAY_IP through a drop-in configuration snippet in /etc/anon-ws-disable-stacked-tor.d configuration folder instead.
```

Comment:
I believe this comment should reference `GATEWAY_IP6` here.

Reply:
Good catch, fixed.